### PR TITLE
docs: document kernel requirements

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
Add section to install document with some kernel requirements for
the different gadgets.

Fixes https://github.com/kinvolk/inspektor-gadget/issues/233

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -74,3 +74,32 @@ The different supported modes can be set by using the `hook-mode` option:
...
+run the gadgets. This has been generated by testing different Ubuntu versions
+and their shipped kernels, hence it's possible that some gadgets works in older
+kernels than the one mentioned here.
+
+
+| Gadget                 | Tools Mode | Required Kernel                 | BTF Required |
+|------------------------|------------|---------------------------------|--------------|
+| bindnsoop              | standard   | 4.15                            | N            |
+| bindnsoop              | core       | 5.4<sup>[1](#btfhubnote)</sup>  | Y            |
+| capabalities           |            | 4.15                            | N            |
```
Comment (alban) [inline docs/install.md:90]:
```suggestion
| capabilities           |            | 4.15                            | N            |
```

File: docs/install.md [hunk 2]
```diff
@@ -74,3 +74,32 @@ The different supported modes can be set by using the `hook-mode` option:
...
+| dns                    |            | 5.4                             | N            |
+| execsnoop              | standard   | 4.15                            | N            |
+| execsnoop              | core       | 5.4<sup>[1](#btfhubnote)</sup>  | Y            |
+| network policy advisor |            |                                 |              |
```
Comment (alban) [inline docs/install.md:94]: Did you have any kernel version for this one?

Comment (mauriciovasquezbernal) [reply docs/install.md:94]: I haven't gathered the data for that one. I just also realized I'm missing the seccomp policy advisor too.

Comment (alban) [reply docs/install.md:94]: I think it's ok to merge this first version and then improve with the missing information in  another PR.

**Review by alban — APPROVED** [review]

**Comment by blanquicet** [comment]
This PR completed a task in the TODO list of PR https://github.com/kinvolk/inspektor-gadget/pull/227.

**Pull request merged**